tests/test_modules.py

```diff
@@ -101,3 +102,15 @@ def func(x):
         for f in info['pkg_ver_list']:
             print(f[0])
 
+class InteractiveTest(unittest.TestCase):
+    def setUp(self):
+        self.wrenexec = pywren.default_executor()
+
+    def test_simple(self):
+        ret = os.system("python -c \"import pywren;import extmodule;pwex = pywren.default_executor();results = pwex.map(extmodule.foo_add, [0]);print(results[0].result())\"")
```

Comment:
According to the python docs: "The subprocess module provides more powerful facilities for spawning new processes and retrieving their results; using that module is preferable to using this function. See the Replacing Older Functions with the subprocess Module section in the subprocess documentation for some helpful recipes."

Can we try using `subprocess` instead?